container/go/pkg/compat/path.go

```diff
@@ -0,0 +1,58 @@
+/// Copyright 2015 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//////////////////////////////////////////////////////////////////////
+// Path utils used for legacy image layout outputted by python containerregistry.
+// Uses the go-containerregistry API as backend.
+
+package compat
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// Expected metadata files in legacy layout.
+const manifestFile string = "manifest.json"
+const configFile string = "config.json"
+const digestFile string = "digest"
+
+// Path returns a full directory of this path concatenated with other <elem> paths.
+func Path(path string, elem ...string) string {
```

Comment:
Don't see a need for this function. Just doing filepath.Join wherever this is called should suffice as this function is not really doing anything special to handle docker images.